packages/ppx_default/ppx_default.0.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Generate default values for your types"
+maintainer: ["franciscosantosfcp@hotmail.com"]
+authors: ["Francisco Santos"]
+license: "MIT"
+homepage: "https://github.com/ProgramingIsTheFuture/ppx_default"
+bug-reports: "https://github.com/ProgramingIsTheFuture/ppx_default/issues"
+depends: [
+  "dune" {>= "3.10"}
+  "ocaml" {>= "4.14"}
+  "ppxlib" {>= "0.25"}
+  "ppx_deriving"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ProgramingIsTheFuture/ppx_default.git"
+url {
+  src:
+    "https://github.com/ProgramingIsTheFuture/ppx_default/releases/download/v0.1.1/ppx_default-v0.1.1.tbz"
+  checksum: [
+    "sha256=73f86c412c3093fc72f3ba17b812348ac7b4a6d9c46e47f196c218d53d19619b"
+    "sha512=8a5cb8085f16dacc3d779a7e31045a6b8cdf404acc55e82f45679117081be1b0e30b30f47ecc07241ad771edf1b47884423e6179eb7401204657c652fadb77fd"
```

Comment:
You might need to update the checksums after changing the tagged release. CI is giving the following error and refusing to build:

```
ppx_default.0.1.1 (failed: Failed to get sources of ppx_default.0.1.1: Bad checksum)
```